doc/examples/tracking_tissue_classifier.py

```diff
@@ -0,0 +1,319 @@
+"""
+================================================
+Using Various Tissue Classifier for Tractography
+================================================
+The tissue classifier determines if the tracking stops or continues at each
+tracking position. If the tracking stops, it can either be because it reaches
+an ending region (e.g. low FA, gray matter or corticospinal fluid regions),
+because it exit the image boundary or because the direction getter has no
+direction to follow. Each tissue classifier determines if the stoping is 'valid'
+or 'invalid'. A streamline is 'valid' when the tissue classifier determines the
+streamline stops in a position classified as 'ENDPOINT' or 'OUTSIDEIMAGE'. A
+streamline is 'invalid' when it stops in a position classified as 'TRACKPOINT'
+or 'INVALIDPOINT'. These conditions are described below. The 'LocalTracking'
+generator can be set to output all generated streamlines or only the 'valid'
+ones.
+
+This example is an extension of the
+:ref:``example_deterministic_fiber_tracking`` example. We begin by loading the
+data, fitting a constant solid angle (CSA) reconstruction
+model and creating the maximum determnistic direction getter.
+"""
+
+import numpy as np
+
+from dipy.data import read_stanford_labels, default_sphere
+from dipy.direction import DeterministicMaximumDirectionGetter
+from dipy.io.trackvis import save_trk
+from dipy.reconst.shm import CsaOdfModel
+from dipy.tracking.local import LocalTracking
+from dipy.tracking import utils
+from dipy.viz import fvtk
+from dipy.viz.colormap import line_colors
+
+ren = fvtk.ren()
+
+hardi_img, gtab, labels_img = read_stanford_labels()
+data = hardi_img.get_data()
+labels = labels_img.get_data()
+affine = hardi_img.get_affine()
+
+seed_mask = labels == 2
+white_matter = (labels == 1) | (labels == 2)
+seeds = utils.seeds_from_mask(seed_mask, density=2, affine=affine)
+
+csa_model = CsaOdfModel(gtab, 4)
```

Comment:
But I don't want promoting a method for reconstruction that we know is not as good as CSD.